.github/workflows/test-python.yaml

```diff
@@ -49,28 +52,29 @@ jobs:
         with:
           name: wheels
           path: dist
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.7"
       - uses: actions/setup-python@v4
         with:
           python-version: "3.8"
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.9"
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.10"
-      - name: Install nox
-        run: pipx install nox
-        shell: bash
-      - name: Cache Nox
-        uses: actions/cache@v3
-        with:
-          path: .nox
-          key:
-            nox-${{ hashFiles('prql-python/noxfile.py') }}-${{
-            hashFiles('prql-python/requirements.txt') }}
-      - name: Run tests
-        shell: bash
-        run: nox -s tests -f bindings/prql-python/noxfile.py
+      - run: pip install maturin
+      - name: Build wheel
+        run: task build-python-wheel
```

Comment:
The wheels have already been built.

https://github.com/PRQL/prql/blob/8d2a55d23a81e17c8fd9a632a267ae95f21083a3/.github/workflows/test-python.yaml#L16-L44